src/pages/Invoice/ListInvoices.js

```diff
@@ -1,7 +1,115 @@
-import React from "react";
+import React, { Fragment, useEffect, useState, useCallback } from "react";
+import withWrapper from "common/withWrapper";
+import Navbar from "common/Navbar";
+import { Link } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import "styles/add-customer.css";
+import { useNotification } from "notification";
 
 const ListInvoices = () => {
-    return <div>I am list invoice page</div>;
+    const [tableData, setTableData] = useState([]);
+    useEffect(() => {
+        fetchData();
+    }, []);
+
+    const { triggerNotification } = useNotification();
+    // Function to fetch data from local storage
+    const fetchData = useCallback(() => {
+        if (localStorage.getItem("invoice_data")) {
+            try {
+                const invoiceData = JSON.parse(
+                    localStorage.getItem("invoice_data")
+                );
+                setTableData(invoiceData);
+            } catch (e) {
+                triggerNotification("Failed parsing inventory data", {
+                    type: "error",
+                });
+                localStorage.removeItem("invoice_data");
+            }
+        }
+    }, [tableData]);
+
+    // Function to calc total
+    const calcAmount = (array) => {
+        if (array) {
+            return array.reduce((accumulator, currValue) => {
+                return accumulator + currValue.quantity * currValue.price;
+            }, 0);
+        }
+        return 0;
+    };
+
+    return (
+        <Fragment>
+            <Navbar opened="invoice" />
```

Comment:
Move Navbar to App.js or index.js or wrapper
use location path to decide which route is active.